testsuite/tests/prometheus/system/test_internal_calls.py

```diff
@@ -0,0 +1,118 @@
+"""
+Test Prometheus metric for content_caching.
+"""
+import base64
+import time
+
+import pytest
+import requests
+
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+from testsuite import TESTED_VERSION, rawobj  # noqa # pylint: disable=unused-import
+from testsuite.prometheus import PROMETHEUS_REFRESH
+
+pytestmark = [
+    # can not be run in parallel
```

Comment:
this is insufficient, one line comment is great, however it must be explanatory

Reply:
This comment is not accurate in this case, will remove it.